content/desktop/release-notes.md

```diff
@@ -24,6 +24,75 @@ Take a look at the [Docker Public Roadmap](https://github.com/docker/roadmap/pro
 
 For frequently asked questions about Docker Desktop releases, see [FAQs](faqs/releases.md).
 
+## 4.26.0
+
+{{< release-date date="2023-11-30" >}}
+
+{{< desktop-install all=true version="4.26.0" build_path="/TBD/" >}}
+
+### New
+
+- Administrators can now control access to beta and experimental features in the **Features in development** tab with [Settings Management](hardened-desktop/settings-management/configure.md).
+- Introduced four new version update states in the footer.
+- `docker init` (Beta) now supports PHP with Apache + Composer.
+- The [**Builds** view](use-desktop/builds.md) is now GA. You can now inspect builds, troubleshoot errors, and optimize build speed.
+
+### Upgrades
+
+- [Compose v2.23.3](https://github.com/docker/compose/releases/tag/v2.23.3)
+- [Docker Scout CLI v1.2.0](https://github.com/docker/scout-cli/releases/tag/v1.2.0).
+- [Buildx v0.12.0](https://github.com/docker/buildx/releases/tag/v0.12.0)
+- [Wasm](../desktop/wasm/_index.md) runtimes:
+  - wasmtime, wasmedge and wasmer `v0.3.1`.
+  - lunatic, slight, spin, and wws `v0.10.0`.
+  - Wasmtime is now based on wasmtime `v14.0` and supports wasi preview-2 components
+  - Wasmedge is now based on WasmEdge `v0.13.5`
+  - Spin is now based on Spin `v2.0.1`
+  - wws is now based on wws `v1.7.0`
+- [Docker Engine v24.0.7](https://docs.docker.com/engine/release-notes/24.0/#2407)
+- [Containerd v1.6.25](https://github.com/containerd/containerd/releases/tag/v1.6.25)
+- [runc v1.1.10](https://github.com/opencontainers/runc/releases/tag/v1.1.10)
+
+### Bug fixes and enhancements
+
+#### For all platforms
+
+- You can now provide feedback from the commandline by using `docker feedback`.
+- Improved the text and position of the startup options in the **General** settings tab.
```

Comment:
Proposing a line for for design system changes (wording discussed and agreed within frontend platform team):

```suggestion
- Improved the text and position of the startup options in the **General** settings tab.
- Redesigned the dashboard's header bar, added links to other Docker resources, improved display of account information.
```